commit 6f228ea26be06572de245aed5496aaa122cca5a8
Author: Steve Dickson <steved@redhat.com>
Date:   Fri Oct 15 17:20:28 2010 -0400

    Removed duplicate entries in export man page
    
    The man page's paragraphs about "refer=" and "replicas="
    each appear twice.
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/exportfs/exports.man b/utils/exportfs/exports.man
index c726dd9..bfc4d42 100644
--- a/utils/exportfs/exports.man
+++ b/utils/exportfs/exports.man
@@ -375,20 +375,6 @@ If the client asks for alternative locations for the export point, it
 will be given this list of alternatives. (Note that actual replication
 of the filesystem must be handled elsewhere.)
 
-.TP
-.IR refer= path@host[+host][:path@host[+host]]
-A client referencing the export point will be directed to choose from
-the given list an alternative location for the filesystem.
-(Note that the server must have a mountpoint here, though a different
-filesystem is not required; so, for example,
-.IR "mount --bind" " /path /path"
-is sufficient.)
-.TP
-.IR replicas= path@host[+host][:path@host[+host]]
-If the client asks for alternative locations for the export point, it
-will be given this list of alternatives. (Note that actual replication
-of the filesystem must be handled elsewhere.)
-
 .SS User ID Mapping
 .PP
 .B nfsd
